lib/api/api.js

```diff
@@ -105,6 +106,7 @@ const api = {
     bucketGetCors,
     bucketGetVersioning,
     bucketGetWebsite,
+    bucketGetLocation,
```

Comment:
The arsenal requestContext actionMap needs to be updated to handle this new api.
See here: https://github.com/scality/Arsenal/blob/master/lib/policyEvaluator/RequestContext.js#L10
AWS uses 's3:GetBucketLocation' as the action.  See: http://docs.aws.amazon.com/AmazonS3/latest/dev/using-with-s3-actions.html

We should also add some policy tests to integration with this api.